bindings/pyroot/cppyy/CPyCppyy/src/MemoryRegulator.cxx

```diff
@@ -244,3 +244,16 @@ PyObject* CPyCppyy::MemoryRegulator::RetrievePyObject(Cppyy::TCppObject_t cppobj
 
     return nullptr;
 }
+
+
+//-----------------------------------------------------------------------------
+void CPyCppyy::MemoryRegulator::SetRegisterHook(MemHook_t h) {
```

Comment:
I saw these changes (and the ones in the header) are now in upstream cppyy, nice!

I believe this is the only thing you needed to contribute with upstream?

Reply:
> I saw these changes (and the ones in the header) are now in upstream cppyy, nice!
> 
> I believe this is the only thing you needed to contribute with upstream?

Thanks! And yes, it's the only requirement I had